### PR TITLE
feat(dev-console): added feedback spa link in feedback list page

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Fix

# Explain the feature/fix

1. Added feedback spa link to feedback list page in dev console
2. Truncated summary in table to 200 characters
3. Added color change for bug in list view

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

## Feedback List

Screenshot 2022-02-09 at 12 00 43 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

File: packages/developer-console-spa/src/components/feedback/FeedbackList.tsx [hunk 1]
```diff
@@ -46,14 +46,14 @@ type FeedbackState = {
 
 const SORT_SELECT_OPTIONS = [
   {
-    value: 'CREATED_ON',
-    toString: () => 'Sort by: newest'
+    value: "CREATED_ON",
+    toString: () => "Sort by: newest",
```
Comment (riginoommen) [inline packages/developer-console-spa/src/components/feedback/FeedbackList.tsx:50]:
```suggestion
    value: "CREATED_ON",
    toString: () => 'Sort by: newest',
```

File: packages/developer-console-spa/src/components/feedback/FeedbackList.tsx [hunk 2]
```diff
@@ -46,14 +46,14 @@ type FeedbackState = {
 
 const SORT_SELECT_OPTIONS = [
   {
-    value: 'CREATED_ON',
-    toString: () => 'Sort by: newest'
+    value: "CREATED_ON",
```
Comment (riginoommen) [inline packages/developer-console-spa/src/components/feedback/FeedbackList.tsx:49]:
```suggestion
    value: 'CREATED_ON',
```

File: packages/developer-console-spa/src/components/feedback/FeedbackList.tsx [hunk 3]
```diff
@@ -46,14 +46,14 @@ type FeedbackState = {
...
+    toString: () => "Sort by: newest",
   },
-   {
-    value: 'UPDATED_ON',
-    toString: () => 'Sort by: activity'
+  {
+    value: "UPDATED_ON",
+    toString: () => "Sort by: activity",
```
Comment (riginoommen) [inline packages/developer-console-spa/src/components/feedback/FeedbackList.tsx:54]:
```suggestion
    value: 'UPDATED_ON',
    toString: () => 'Sort by: activity',
```

File: packages/developer-console-spa/src/components/feedback/FeedbackList.tsx [hunk 4]
```diff
@@ -111,6 +111,11 @@ export const FeedbackList = () => {
     setSearch(value);
   };
 
+  const truncateText = useCallback(
+    (text: string) => (text.length > 200 ? text.slice(0, 200) + "..." : text),
```
Comment (riginoommen) [inline packages/developer-console-spa/src/components/feedback/FeedbackList.tsx:115]:
```suggestion
    (text: string) => (text.length > 200 ? text.slice(0, 200) + '...' : text),
```

**Review by riginoommen — CHANGES_REQUESTED** [review]
Quotes selections should be consistent

**Comment by akhilmhdh** [comment]
@riginoommen Sorry about that. But its a vscode issue. We are having so much collision going on in linting.

There is editor.config, vs code workspace, then another set of base eslint and prettier issues.

But yah fixed it

**Review by riginoommen — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]
LGTM

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**